src/components/widgets/thermals/TemperatureTargets.vue

```diff
@@ -187,7 +187,12 @@
                 <span
                   v-bind="attrs"
                   v-on="on"
-                >{{ item.temperature.toFixed(1) }}<small>°C</small></span>
+                >{{ item.temperature.toFixed(1) }}
+                  <small>°C
```

Comment:
The newline before this introduces an issue where the `°C` label wraps to a new line (because newlines in HTML act like spaces that turn into wrap-points):
![image](https://user-images.githubusercontent.com/25269274/159092920-f69dcf48-5545-4225-ba7b-bfc8647e5a69.png)

I would suggest creating 2 `<small>` elements (or keeping the `°C` on the same line as the temperature):
```vue
<span
   v-bind="attrs"
  v-on="on"
>{{ item.temperature.toFixed(1) }}<small>°C</small>
  <small>
    <span><br>{{ (item.humidity || 1).toFixed(1) }}&nbsp;%</span>
    <span><br>{{ (item.pressure || 1).toFixed(1) }}&nbsp;hpa</span>
  </small>
</span>
```

Reply:
Nice catch! I did some changes that should now fix this issue!